src/config/utils.ts

```diff
@@ -129,21 +129,29 @@ export const LayerDefinitions: LayersMap = (() => {
   return layers;
 })();
 
-export function getBoundaryLayerSingleton(): BoundaryLayerProps {
+export function getBoundaryLayers(): BoundaryLayerProps[] {
   const boundaryLayers = Object.values(LayerDefinitions).filter(
     (layer): layer is BoundaryLayerProps => layer.type === 'boundary',
   );
-  if (boundaryLayers.length > 1) {
+  if (boundaryLayers.length === 0) {
     throw new Error(
-      'More than one Boundary Layer defined! There should only be one boundary layer in layers.json',
+      'No Boundary Layer found! There should be exactly one boundary layer defined in layers.json.',
```

Comment:
This is no longer true.

Reply:
Suggested alternative text:
"No boundary layer found. There should be at least one boundary layer defined in layers.json"